pages/edge-services/troubleshooting/cache-problems.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: I am experiencing problems with my Edge Services cache
+description: Troubleshoot issues that may arise with caching on your Edge Services pipeline.
+tags: edge-services cache cookies load-balancer
+dates:
+  validation: 2025-11-05
+  posted: 2025-11-05
+---
+
+You may experience unexpected behavior with caching on your Edge Services pipeline. 
+
+This page helps you resolve common problems.
+
+<Message type="tip">
+Before diving into this troubleshooting guide, ensure you are correctly using `curl` for your testing. `curl -I` uses the HEAD method by default, which does not always trigger the same logic as a real `GET` request. This can mislead you during cache tests, as cache checks might be skipped entirely, and responses main contain different headers. Use `curl -X GET -I` for more better testing and accurate `X-Cache` header information in responses.
```

Comment:
```suggestion
Before diving into this troubleshooting guide, ensure you are correctly using `curl` for your testing. `curl -I` uses the `HEAD` method by default, which does not always trigger the same logic as a real `GET` request. This can mislead you during cache tests, as cache checks might be skipped entirely, and responses may contain different headers. Use `curl -X GET -I` for better testing and more accurate `X-Cache` header information in responses.
```